src/core/Val.ml

```diff
@@ -906,7 +887,7 @@ struct
             Abs.make1 @@ fun w -> ext_apply (cdr (fixer_fiber phi)) [`Atom w]
           in
           let el1 = make_hcom (Dir.make `Dim1 r') info.ty1 (base I.idn r r') @@
-            force_abs_sys [face0; face1; face_diag; face_front]
+            force_abs_sys [face0; face_diag; face_front]
```

Comment:
So, let me make sure I understand. The `face1` was `r=1`, making the entire old cofibration `r=0 \/ r=1 \/ r=r' \/ r'=0`. Now the cofibration is `r=0 \/ r=r' \/ r' = 0`; and this becomes valid under the new order because we can also take into account the direction `1~>r'`?

Reply:
It is valid for one of the two reasons: Including the direction makes it valid. Bipartiteness checking also makes it valid.